### PR TITLE
Mandos  - "scQuery" step

### PR DESCRIPTION
**Pull request by andrei-marinica**
Mandos can simulate SC queries.
Also updates tests

### CODE REVIEW

**Review by camilbancioiu — APPROVED** [review]

File: mandos-go/json/model/transaction.go [hunk 1]
```diff
@@ -10,6 +10,9 @@ const (
 	// ScCall describes a regular smart contract call
 	ScCall
 
+	// ScQuery describes a regular smart contract call
```
Comment (dorin-iancu) [inline mandos-go/json/model/transaction.go:13]: Update comment: call -> query

File: test/features/mandos/get_caller.scen.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+    "name": "count ones",
+    "comment": "should fail if the processor doesn't support the `count ones` instruction",
```
Comment (dorin-iancu) [inline test/features/mandos/get_caller.scen.json:3]: Update name and comment

**Review by dorin-iancu — APPROVED** [review]
Not sure what's up with the empty wasm files?

**Comment by andrei-marinica** [comment]
> Not sure what's up with the empty wasm files?

I think it's just a github issue, they are non-empty and well on my machine, and they are also run in the tests.

**Review by claudiu725 — APPROVED** [review]

File: mandos-go/json/model/transaction.go (hunk 1 above)
Comment (andrei-marinica) [reply mandos-go/json/model/transaction.go:13]: ok, next PR

File: test/features/mandos/get_caller.scen.json (hunk 2 above)
Comment (andrei-marinica) [reply test/features/mandos/get_caller.scen.json:3]: ok, next PR

**Pull request merged**